sdk/servicebus/Azure.Messaging.ServiceBus/tests/ServiceBusLiveTestBase.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core.Testing;
+
+namespace Azure.Messaging.ServiceBus.Tests
```

Comment:
I'd really, really like to see us try to keep tests completely self-contained without trying to use a bunch of shared constructs.  That typically causes a ton of complexity to save a very small bit of repetition.    The more straight-forward and self-contained these are, the better the community experience for contributing will be.

Reply:
Not sure I follow. Do you mean we shouldn't have inheritance for test classes? This is useful for methods like GetRandomBuffer, as well as common code needed for resource creation. For instance, I would think the code for creating a Queue would live in ServiceBusLiveTestBase.